ios/FluentUI.Demo/FluentUI.Demo/Demos/ContactViewDemoController.swift

```diff
@@ -32,12 +32,16 @@ class ContactViewDemoController: DemoController {
         let phoneNumberContactView = ContactView(identifier: "+1 (425) 123 4567")
         phoneNumberContactView.translatesAutoresizingMaskIntoConstraints = false
 
+        // ContactView with no title, subtitle or identifier
+        let emptyContactView = ContactView(title: "", subtitle: "")
+        emptyContactView.translatesAutoresizingMaskIntoConstraints = false
+
         addRow(text: "ContactView with image", items: [contactView], textWidth: 200)
         addRow(text: "ContactView with initials", items: [noImageContactView], textWidth: 200)
         addRow(text: "ContactView with first name", items: [firstNameContactView], textWidth: 200)
         addRow(text: "ContactView with email", items: [emailContactView], textWidth: 200)
         addRow(text: "ContactView with phone number", items: [phoneNumberContactView], textWidth: 200)
-        container.addArrangedSubview(UIView())
+        addRow(text: "ContactView with nothing", items: [emptyContactView], textWidth: 200)
```

Comment:
nit: "Empty ContactView" 
"with nothing" sounds a bit depressing 😆

Reply:
🌞  Will try to brighten up the code 🌞